test/src/module/storage/s3Test.c

```diff
@@ -57,14 +57,13 @@ testRequest(IoWrite *write, Storage *s3, const char *verb, const char *path, Tes
     {
         strCatFmt(
             request,
-                "authorization:AWS4-HMAC-SHA256 Credential=%s/\?\?\?\?\?\?\?\?/us-east-1/s3/aws4_request,"
-                    "SignedHeaders=content-length",
-                param.accessKey == NULL ? strZ(driver->accessKey) : param.accessKey);
+            "authorization:AWS4-HMAC-SHA256 Credential=%s/\?\?\?\?\?\?\?\?/us-east-1/s3/aws4_request,SignedHeaders=",
+            param.accessKey == NULL ? strZ(driver->accessKey) : param.accessKey);
 
```

Comment:
It's a bit unclear to me that we are proving we do not sign the Content-Length:0 header, as was the ask, in this test file. Maybe you could point me to where we test this?

Reply:
content-length was in the signed headers test list before, but now it is not. What's unclear?

Reply:
I know, but I was looking for when you call storageS3Auth with HttpHeader *httpHeader that includes content-length - but since you must be getting coverage on the new code in that function, it must be somewhere, so OK.

Reply:
Oh, I see. Every request contains content-length so it definitely gets tested. If you look at s3Test line 75 you can see that the header is tested unconditionally. We just removed it from the signed headers list.